pulp_rpm/app/models/repository.py

```diff
@@ -102,6 +108,74 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+class UlnRemote(Remote):
+    """
+    Remote for "uln" content.
+    """
+
+    TYPE = "uln"
+
+    uln_user = models.CharField(max_length=512, null=True)
+
+    uln_pwd = models.CharField(max_length=512, null=True)
```

Comment:
From other conversations on this PR, it seems like uln_user and uln_pwd are required, should null=True be removed?

Reply:
I am still unsure if it would be possible/better to reuse the `username`/`password` fields from the `pulpcore` `Remote` base class instead of adding these additional fields. Can anyone confirm for me if those fields have a separate dedicated use? I would like to avoid having two sets of username and password fields if possible.

Reply:
I think it would be better to use username and password. The DownloaderFactory can read the data from those fields and you'd need to have DownloaderFactory read from `uln_user` and `uln_pwd` anyway so I don't think it's additional work. Introducing new fields I think would only make sense if users would meaningfully set `username`, `password` and `uln_user`, and `uln_pwd` all at the same time which I think is not the case.

Reply:
Done.